examples/next-openai/app/api/use-chat-tools/route.ts

```diff
@@ -10,6 +11,7 @@ export async function POST(req: Request) {
 
   const result = streamText({
     model: openai('gpt-4o'),
+    // model: anthropic('claude-3-5-sonnet-latest'),
```

Comment:
stray comment

Reply:
wanted to leave it in for future testing